config.default.json

```diff
@@ -80,6 +83,24 @@
         "BROKER_CLIENT_URL": "https://<broker.client.hostname>:<port>"
       }
     },
+    "apprisk-sonarqube": {
```

Comment:
i think should also be under `apprisk` the key is the integration type and we have one integration of type `apprisk` for all the 3rd party integrations (at least this is what we decided when first implemented)
@aarlaud keep me honest please